CMake/ExternalData.cmake

```diff
@@ -999,7 +999,7 @@ function(_ExternalData_get_from_object_store hash algo var_obj var_success)
   foreach(dir ${ExternalData_OBJECT_STORES})
     set(obj "${dir}/${algo}/${hash}")
     if(EXISTS "${obj}")
-      message(STATUS "Found object: \"${obj}\"")
+      # message(STATUS "Found object: \"${obj}\"")
```

Comment:
If you think these messages are noisy and redundant, please just remove them! Because I think commented out code is noisy as well!

Reply:
Thanks. Done.